Brewfile

```diff
@@ -1,4 +1,3 @@
 brew "docker"
-brew "protobuf"
```

Comment:
It was necessary to compile the OpenTelemetry OTLP exporter in the binary engine, which we didn't need for a long time even when we were still using OpenTelemetry in the engines generally, and this PR removes OTel in favor of Tokio `tracing` from the engines completely.